website/blog/2023-01-18-release-0.11.md

```diff
@@ -0,0 +1,256 @@
+
+When starting/stopping or deleting a container, a spinner is now displayed. In case of error, a message indicating that the action failed will also be better indicated. 
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/436777/211531610-2347d302-4918-46ae-a5a2-c80fac0314f5.mp4' />
+
+For containers that exit immediately or short-lived containers, the feedback is also improved and include report of error now provide a better feedback to the user [#1161](https://github.com/containers/podman-desktop/pull/1161).
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/436777/211831905-ebf596d5-efc8-4f55-8cb8-3f31655388b9.mp4' />
+
+
+#### Allows to change the default font size for the editor [#1160](https://github.com/containers/podman-desktop/pull/1160)
+
+The editors used is different screen of Podman Desktop, from the inspect screen, to container's outputs and Kubernetes YAML. The default font size is 10 pixels. It's now possible to adjust the font size to the one the one you prefer. This setting is persisted and will be used for all the editors of Podman Desktop and available from the preferences page (Settings -> Preferences).
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/436777/211778161-130ff733-b2ca-4306-bea3-d031196c3b29.mp4' />
+
+#### Keep expanded state of pods when refreshing containers [#1042](https://github.com/containers/podman-desktop/pull/1042)
+
+When switching from different screens of the application or simply refreshing the list of containers, the expanded state of each item in the list is now persisted and will be properly restored.
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/436777/207864147-b68ea9bd-0ca9-42dc-882e-b8a705233749.mp4' />
+
+#### Click on the Pod name redirects to the Pod details page [#1159](https://github.com/containers/podman-desktop/pull/1159)
+
+The list of containers displays pods which may also be running as well. Now clicking on the pod name directly redirects to the Pod details page.
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/436777/211770946-2255f39f-7e2e-48ad-9ead-bcbfe6a115a7.mp4' />
+
+#### Improved styles of buttons for actions [#984](https://github.com/containers/podman-desktop/pull/984)
+
+The style of the buttons for actions on item in the list of in details pages have been improved. The background has been removed, but to make the hover state more visible, the "hover" circle is visble and the icon's color is also changing.
+
+On lists:
+![list-actions](https://user-images.githubusercontent.com/6422176/205979121-b49a0ddf-03bb-4a4d-8d12-bc8d0bd52387.png)
+
+On details pages:
+![details-actions](https://user-images.githubusercontent.com/6422176/205979123-ea420b17-e834-4029-82eb-22949889eee9.png)
+
+#### Improved alignments in pages with lists [#1182](https://github.com/containers/podman-desktop/pull/1182)
```

Comment:
As much as I love my PR getting called out ;-) , in the future we could probably summarize something this minor to "Improved column alignment" in the Notable list.

Reply:
:)